spec/javascript/components/molecules/StatusDropdown.test.js

```diff
@@ -0,0 +1,130 @@
+import { LEASE_UP_STATUS_OPTIONS } from '~/utils/statusUtils'
+
+const ON_CHANGE = jest.fn()
+
+const getWrapper = (propOverrides = {}) => mount(
+  <StatusDropdown
+    onChange={ON_CHANGE}
+    {...propOverrides}
+  />
+)
+
+describe('StatusDropdown', () => {
+  let wrapper
+  test('it renders a select', () => {
+    wrapper = getWrapper()
+    expect(wrapper.find('Select').exists()).toBeTruthy()
+  })
+
+  test('it renders with default props correctly', () => {
+    wrapper = getWrapper()
+
+    const dropdownProps = wrapper.find('Dropdown').props()
+    expect(dropdownProps.items).toEqual(LEASE_UP_STATUS_OPTIONS)
+    expect(dropdownProps.value).toBeNull()
+    expect(dropdownProps.placeholder).toEqual('Status')
+    expect(dropdownProps.disabled).toBeFalsy()
+  })
+
+  describe('it renders the toggle as expected', () => {
+    test('when value is null', () => {
+      wrapper = getWrapper()
+      const expectedButtonClasses = [
+        'button',
+        'dropdown-button',
+        'has-icon--right',
+        'text-align-left',
+        'tertiary'
+      ]
+      const toggleButton = wrapper.find('Select').find('.status-dropdown__control').find('button')
+      expect(toggleButton.text()).toEqual('Status')
+      expect(toggleButton.hasClass(expectedButtonClasses.join(' '))).toEqual(true)
+      expect(toggleButton.prop('disabled')).toEqual(false)
+    })
+
+    test('without any additional button styles', () => {
+      wrapper = getWrapper()
+      const toggleButton = wrapper.find('Select').find('.status-dropdown__control').find('button')
+      expect(toggleButton.hasClass('expand')).toBe(false)
+      expect(toggleButton.hasClass('tiny')).toBe(false)
+      expect(toggleButton.hasClass('small')).toBe(false)
+    })
+
+    test('when additional styles are provided', () => {
+      wrapper = getWrapper({ expand: true, size: 'tiny' })
+      let toggleButton = wrapper.find('Select').find('.status-dropdown__control').find('button')
+      expect(toggleButton.hasClass('expand')).toBe(true)
+      expect(toggleButton.hasClass('tiny')).toBe(true)
+      expect(toggleButton.hasClass('small')).toBe(false)
+
+      wrapper = getWrapper({ expand: true, size: 'small' })
+      toggleButton = wrapper.find('Select').find('.status-dropdown__control').find('button')
+      expect(toggleButton.hasClass('tiny')).toBe(false)
+      expect(toggleButton.hasClass('small')).toBe(true)
+    })
+
+    test('when value provided and is styled', () => {
+      wrapper = getWrapper({ status: 'Appealed' })
+      const expectedButtonClasses = [
+        'button',
+        'dropdown-button',
+        'has-icon--right',
+        'text-align-left',
+        'is-appealed'
+      ]
+      const toggleButton = wrapper.find('Select').find('.status-dropdown__control').find('button')
+      expect(toggleButton.text()).toEqual('Appealed')
+      expect(toggleButton.hasClass(expectedButtonClasses.join(' '))).toBe(true)
```

Comment:
I think this might be tied a little closely to the implementation, since I think this will check ordering too.

How about we just check that the is-appealed class is added here, since that seems most relevant to the test